app/rdf/query-search.ts

```diff
@@ -151,13 +153,26 @@ export const searchCubes = async ({
           locale,
         })}
       }
+      OPTIONAL {
+        ?iri ${ns.schema.about} ?subthemeIri .
+        OPTIONAL {
+          ?subthemeIri ${
+            ns.schema.inDefinedTermSet
+          } <https://register.ld.admin.ch/foen/theme> .
+        }
+        ${buildLocalizedSubQuery(
+          "subthemeIri",
+          "schema:name",
+          "subthemeLabel",
+          { locale }
+        )}
+      }
```

Comment:
Why moving it here ?

Reply:
It was moved here to avoid a need for an additional query; basically to fetch everything related to cube metadata directly inside this one query